### PR TITLE
Do not install spacewalk-utils-extra on server image

### PR DESCRIPTION
**Pull request by rjmateus**
## What does this PR change?

Do not install the spacewalk-utils-extra on the server image. At the same time move the taskotop tool to the spacewalk-utils package.

Next step would be remove the package `spacewalk-utils-extra`

## GUI diff

No difference.

Before:

After:

- [ ] **DONE**

## Documentation
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- No documentation needed: only internal and user invisible changes
- Documentation issue was created: [Link for SUSE Manager contributors](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31), [Link for community contributors](https://github.com/uyuni-project/uyuni-docs/issues/new).
- API documentation added: please review the Wiki page [Writing Documentation for the API](https://github.com/uyuni-project/uyuni/wiki/Writing-documentation-for-the-API) if you have any changes to API documentation.
- (OPTIONAL) [Documentation PR](https://github.com/uyuni-project/uyuni-docs/pulls)

- [ ] **DONE**

## Test coverage
ℹ️ If a major new functionality is added, it is **strongly recommended** that tests for the new functionality are added to the Cucumber test suite
- No tests: **add explanation**
- No tests: already covered
- Unit tests were added
- Cucumber tests were added

- [ ] **DONE**

## Links

Issue(s): #
Port(s): # **add downstream PR(s), if any**

Need port to 5.0 branch when it's ready.

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: containers/server-image/Dockerfile [hunk 1]
```diff
@@ -27,7 +27,6 @@ RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-li
     ${PRODUCT_PATTERN_PREFIX}_server \
     ${PRODUCT_PATTERN_PREFIX}_retail \
     billing-data-service \
-    spacewalk-utils-extras \
```
Comment (mcalmer) [inline containers/server-image/Dockerfile:30]: I think there is also a changelog here in server-image

File: utils/spacewalk-utils.changes.rmateus.Manager-4.3 [hunk 2]
```diff
@@ -0,0 +1 @@
+- move takotop tool to spacewalk-utils package
```
Comment (mcalmer) [inline utils/spacewalk-utils.changes.rmateus.Manager-4.3:1]:
```suggestion
- move taskotop tool to spacewalk-utils package
```

**Review by mcalmer — CHANGES_REQUESTED** [review]

**Comment by rjmateus** [comment]
I have tested the PR using the Hexagon project in IBS, and it worked as expected. `spacewalk-utils-extra` was not installed and the tool taskotop was installed as part of `spacewalk-utils`

**Comment by rjmateus** [comment]
lets not do it for now, and we must check each individually tool based on this page: https://confluence.suse.com/display/~RDiasMateus/SUMA+cmd+tools

**Pull request closed without merging**